clusterloader2/pkg/prometheus/manifests/0ssd-storage-class.yaml

```diff
@@ -7,7 +7,18 @@ metadata:
   name: ssd
 provisioner: {{$PROMETHEUS_STORAGE_CLASS_PROVISIONER}}
 parameters:
+  {{if eq $PROMETHEUS_STORAGE_CLASS_PROVISIONER "kubernetes.io/gce-pd"}}
```

Comment:
This stops setting type for PROMETHEUS_STORAGE_CLASS_PROVISIONER != kubernetes.io/gce-pd which is a breaking changes and will break some gke tests.

Can you remove this if and set PROMETHEUS_STORAGE_CLASS_VOLUME_TYPE to empty value when needed?
IIUC empty value shouldn't get overridden by DefaultParam above: https://github.com/kubernetes/perf-tests/blob/d8d8c4b6016376f3871432dd221e44649bb73837/clusterloader2/pkg/config/template_functions.go#L202-L202.

Reply:
`type` is not a valid parameter for azure-disk storageClass and must be replaced with other parameters. I will update this